<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Specviz</class>
 <widget class="QMainWindow" name="Specviz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Specviz</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <widget class="QCustomPlot" name="plotWidget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QMenu" name="menuComplexity">
     <property name="title">
      <string>Complexity</string>
     </property>
     <addaction name="asComplexityLow"/>
     <addaction name="asComplexityMedium"/>
     <addaction name="asComplexityHigh"/>
     <addaction name="asComplexityVeryHigh"/>
    </widget>
    <addaction name="menuComplexity"/>
    <addaction name="separator"/>
    <addaction name="displayFrameAll"/>
    <addaction name="displayFrameSelected"/>
    <addaction name="displayResetView"/>
    <addaction name="separator"/>
    <addaction name="displayExpand"/>
    <addaction name="displayCollapse"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="fileOpen"/>
    <addaction name="separator"/>
    <addaction name="fileExportAll"/>
    <addaction name="fileExportSelected"/>
    <addaction name="separator"/>
    <addaction name="fileExportImage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="helpGithubReadme"/>
    <addaction name="helpGithubIssues"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="editCopyImage"/>
    <addaction name="separator"/>
    <addaction name="editShow"/>
    <addaction name="editHide"/>
    <addaction name="separator"/>
    <addaction name="editDelete"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="DockWidget" name="dockWidget_2">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>469</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QWidget" name="dockWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <widget class="QTreeWidget" name="treeWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
          <widget class="QTreeWidget" name="headerWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="displayFrameAll">
   <property name="text">
    <string>Frame all</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="displayResetView">
   <property name="text">
    <string>Reset view</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="displayFrameSelected">
   <property name="text">
    <string>Frame selected</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="fileOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileExportSelected">
   <property name="text">
    <string>Export selected ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="fileExportImage">
   <property name="text">
    <string>Export image ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="helpGithubReadme">
   <property name="text">
    <string>Open Github README</string>
   </property>
  </action>
  <action name="helpGithubIssues">
   <property name="text">
    <string>Open Github Issues</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="editCopyImage">
   <property name="text">
    <string>Copy image ...</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="fileExportAll">
   <property name="text">
    <string>Export ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="asComplexityLow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Low</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="asComplexityMedium">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="asComplexityHigh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>High</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="asComplexityVeryHigh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Very high</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="editShow">
   <property name="text">
    <string>Show</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="editHide">
   <property name="text">
    <string>Hide</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="editDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="displayExpand">
   <property name="text">
    <string>Expand</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="displayCollapse">
   <property name="text">
    <string>Collapse</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DockWidget</class>
   <extends>QDockWidget</extends>
   <header>../../../sources/dockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../../sources/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="specviz.qrc"/>
 </resources>
 <connections/>
</ui>
